core/client/fs/src/main/java/alluxio/client/block/stream/GrpcBlockingStream.java

```diff
@@ -75,8 +74,8 @@
    * @param bufferSize maximum number of incoming messages the buffer can hold
    * @param description description of this stream
    */
-  public GrpcBlockingStream(Function<StreamObserver<ResT>, StreamObserver<ReqT>> rpcFunc,
```

Comment:
why was this necessary? the constructor now has checked exception.

Reply:
It should have, because the `rpcFunc` is `BlockWorkerClient#readBlock`, which throws checked exception. A `Function` can't throw checked exception in the original code.